src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -635,3 +635,21 @@ def _configure_security_policy_storage_params(cmd):
                options_list=('--end-ip-address',),
                help='The end IP address of the firewall rule. Must be IPv4 format. Use value'
                ' \'0.0.0.0\' to represent all Azure-internal IP addresses.')
+
+
+#####
+#           sql server vnet-rule
+#####
+
+
+with ParametersContext(command='sql server vnet-rule') as c:
+    # Help text needs to be specified because 'sql server firewall-rule update' is a custom
+    # command.
+    c.argument('server_name',
+               options_list=('--server', '-s'))
+
+    c.argument('virtual_network_rule_name',
+               options_list=('--name', '-n'))
```

Comment:
This should use the `name_arg_type`, which will add complete support and should add `--ids` support.

Reply:
Is this for storage consistency?

Reply:
General CLI consistency. If `--ids` isn't appropriate here (i.e. the rule can't be addressed by an ARM id) then at least there should be a completer for the rule name.